storage_service/locations/models/space.py

```diff
@@ -445,7 +445,7 @@ def move_rsync(self, source, destination, try_mv_local=False):
 
         # Rsync file over
         # TODO Do this asyncronously, with restarting failed attempts
-        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw', '-r', source, destination]
+        command = ['rsync', '-t', '-O', '--protect-args', '-vv', '--chmod=ugo+rw,Da+x', '--perms', '-r', source, destination]
```

Comment:
@remileduc, what's the purpose of adding `--perms` when combined with `--chmod`?

I'm also wondering if we should use `--chmod=ug+rw,ug+x` instead, we don't need to give permissions to `others` right? Perhaps easier to read in octal mode: `--chmod=F660,D770`.